### ISSUE TITLE
UX/UI suggestions

### ISSUE THREAD

**Issue by Anton-4** [LemmyNet/joinlemmy-site#159]
Hi,
Thanks for working on Lemmy :heart:

join-lemmy.org is a critical point in convincing people to use Lemmy, and I think some small improvements here can make a big difference. I do a good amount of UX and UI stuff professionally so I hope I can help out with some advice. Naturally, a PR would be most helpful but that requires more time than I can give.

My advice is targeted to increase the chance that the average person would use Lemmy.

"Join a server" is nicely visible, but the user has not been motivated to join a server.
If you check reddit.com, twitter.com and tiktok.com for the first time, you are immediately shown interesting content.
I'd recommend showing interesting posts from your most active communities.

The general language on the website is too technical, short and simple text does wonders.
Some suggestions to this point:
- "A link aggregator for the fediverse." the average person does not know what "link aggregator" or "fediverse" means.
- Use "community" instead of "server".
- The website is too developer oriented.
- I would remove "using the strong copyleft AGPL License.".
- The website is trying to provide information to users and admins at the same time, it's better to split this up in multiple pages. The overwhelming majority of visitors will be users, so that's who you want to focus on.
- The "Blazing Fast", "Powerful Mod Tools", "Create your own discussion platform", "More Features" are not relevant to people who want to view content on Lemmy.

The "Mobile apps for iOS and Android." link is very easy to miss.

The blurred text background increases the business of the page, keep it clean and simple.

Making the Support/Donate section more inviting will boost donations.

I hope this advice is helpful :)

**Comment by sunaurus** [comment]
I strongly agree that the UX could be improved. By far this is the most important point:

> The website is too developer oriented.

I am intending to put together a PR that attempts to address this. In case anybody else is currently working on it or planning to work on it, please join the discussion so we can coordinate!

**Comment by sunaurus** [comment]
One of the key comments I've noticed online from new users is that they get confused when picking a home server. My initial idea to tackle this is to add a F.A.Q. right next to the list of servers on join-lemmy.org. You can see a quick mock-up here:

![image](https://github.com/LemmyNet/joinlemmy-site/assets/5356547/9068169b-235a-437a-9109-18c30b934ea1)

Any thoughts on this from anybody? Would you be interested in a PR that adds this F.A.Q. @Nutomic @dessalines ?

**Comment by fheft** [comment]
I think basically all of those are very good suggestions! Less technical wording, focus on giving a short and easy to grasp introduction to Lemmy and a bit design polishing (that blurred background...).

An FAQ seems like a good idea as well. Maybe we can add some kind of accordion UI to avoid having a huge block of text? Large portions of text can be daunting and maybe even distracting from the list of instances on the right.

**Comment by mispp** [comment]
I would propose an alternative: split the join Lemmy in two parts:
- simple
- advanced

Simple: organize Lemmy instances per country/continent (and reflect this in the domain names e.g. .eu, .us, .lat,...). When non-technical users come to the site and what to join Lemmy, join-lemmy should create an account for them on an instance closest to them. Users should not be asked to select anything instance related in this case. Once Account had been created and user has been logged in, redirect to proper server. This would mean that login should be handled by the main site like join-lemmy and only after a successful login, redirect to the proper/actual server. This applies not only to Account creation, but also to normal operations. Users should not remember which Server they are on if they don't want to know it.

Advanced: list the instances as it is now, with locations and topic specifics so advanced users can pick.

No matter how many faq pages and articles you make, people will just go away because they dont know what to pick. It might not be the biggest sample ever, but most of the people I know do exactly that.
If they do not know/care, do the decision for them. Confirmation step can be added to inform users where their account is being created (with the option to switch) in next-next-next-finish type of account creation process, but this should be possible to ignore/skip with only one 'next'.

**Comment by dessalines** [comment]
We have a new mockup / design for this completed, I just have to code it and build it in tailwind. That keeps getting pushed back due to all the other work lemmy needs. But I will get to it hopefully sometime soon.

**Comment by dessalines** [comment]
You can see the new changes being previewed at http://jointest.lemmy.ml:1234 , bc it has most of these now.

**Comment by Anton-4** [comment]
Nice job! Massive improvement :muscle:

**Comment by calculuschild** [comment]
Looks really nice! Getting this out there will remove a huge hurdle for newcomers.
